lib/mutant/cli.rb

```diff
@@ -155,6 +155,9 @@ def add_filter_options(opts)
       opts.on('--ignore-subject EXPRESSION', 'Ignore subjects that match EXPRESSION as prefix') do |pattern|
         add_matcher(:ignore_expressions, config.expression_parser.(pattern))
       end
+      opts.on('--since REVISION', 'Only select subjects touched since REVISION') do |revision|
```

Comment:
Mhh, I think that git than is a bit inconsistent. Right now as all user fencing documentation calls them `revision`, I think thats the better name.

Actually I like `reference` more as this noun is more closer to the actual meaning. The next time I'll touch this code I'll consider to rename to `refernece`